linkerd/app/profiling/src/bin/profile.rs

```diff
@@ -25,9 +25,11 @@ fn main() {
 
     let srv = server::mock_listening(addr.clone());
     let srv2 = server::mock_listening(addr.clone());
-    let ctrl = controller::new()
-        .destination_and_close("transparency.test.svc.cluster.local", srv.addr)
-        .run();
+
+    let ctrl = controller::new();
+    let transparency_tx = ctrl.destination_tx("transparency.test.svc.cluster.local");
+    transparency_tx.send_addr(srv.addr);
```

Comment:
should we also call `profile_tx_default` here to avoid churning service profile resolutions?